docs/API.md

```diff
@@ -614,7 +614,8 @@ The sidekick configuration.
 | mountpoint | <code>string</code> |  | The content source URL (optional) |
 | project | <code>string</code> |  | The name of the project used in the sharing link (optional) |
 | plugins | [<code>Array.&lt;Plugin&gt;</code>](#Plugin) |  | An array of [plugin configurations](#Plugin) (optional) |
-| outerHost | <code>string</code> |  | The outer CDN's host name (optional) |
+| previewHost | <code>string</code> |  | The host name of a custom preview CDN (optional) |
+| liveHost | <code>string</code> |  | The host name of a custom live CDN (optional) |
```

Comment:
@rofe , are these configurations available in the sidekick? If yes, where?

Reply:
@sdmcraft You can add them to the `/tools/sidekick/config.json`. I forgot to add them to the [schema](https://github.com/adobe/helix-sidekick-extension/blob/main/docs/config.schema.json), will do it now!

Reply:
https://github.com/adobe/helix-sidekick-extension/pull/411